engine/src/main/java/org/terasology/rendering/opengl/fbms/StaticFBM.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.opengl.fbms;
+
+import org.terasology.rendering.opengl.AbstractFBM;
+import org.terasology.rendering.opengl.FBOConfig;
+
+/**
+ * TODO: Add javadocs
+ * TODO: Better naming
+ */
+public class StaticFBM extends AbstractFBM {
```

Comment:
As mentioned earlier, we might not need this. But perhaps it's worth keeping for modules?